cmd/juju/cloud/listcredentials.go

```diff
@@ -58,25 +57,36 @@ A credential for 'controller' model is determined at bootstrap time and
 will be stored on the controller. It is considered to be controller default.
 
 Recall that when a controller is created a 'default' model is also 
-created. This model will use the controller default credential. To see all your
-credentials on the controller use "juju show-credentials" command.
+created. This model will use the controller default credential. 
+To see details of your credentials use "juju show-credentials" command.
```

Comment:
We've aliased show-credential to show-credentials but can we change the text here to use the semantically more correct form of show-credential?